src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -124,14 +121,14 @@
         init() {
             const isEmpty = !this.container.querySelectorAll(SELECTOR_SELECTED_ITEM).length;
 
-            if (isEmpty && this[canSelectOnlyOne]) {
+            if (isEmpty && this[canSelectOnlyOne] && this.hasDefaultSelection) {
                 this[selectFirstItem]();
             }
 
-            this.container.querySelector(SELECTOR_SELECTION_INFO).addEventListener('click', this[onInputClick], false);
+            this.container.querySelector(SELECTOR_SELECTION_INFO).onclick = this[onInputClick];
```

Comment:
Why not `addEventListener `?

Reply:
Because it's easier to reinitialize, when you don't have to call `removeEventListener` and then `addEventListener`.